test/Microsoft.Extensions.Localization.Tests/ResourceManagerStringLocalizerTest.cs

```diff
@@ -59,10 +59,33 @@ public void EnumeratorCacheIsScopedByAssembly()
             Assert.Equal(expectedCallCount, resourceAssembly2.GetManifestResourceStreamCallCount);
         }
 
+        [Fact]
+        public void ResourceManagerStringLocalizer_GetAllStrings_MissingResourceDoesntThrow()
+        {
+            // Arrange
+            var resourceNamesCache = new ResourceNamesCache();
+            var baseName = "testington";
+            var resourceAssembly = new TestAssemblyWrapper("Assembly1");
+            var resourceManager = new TestResourceManager(baseName, resourceAssembly.Assembly);
+            var localizer = new ResourceManagerWithCultureStringLocalizer(
+                resourceManager,
+                resourceAssembly.Assembly,
+                baseName,
+                resourceNamesCache,
+                CultureInfo.CurrentCulture);
+
+            // Act
+            var t = localizer.GetAllStrings(true);
```

Comment:
Because the true case means walking up the list of parents and looking for each of those resources, the failure of which to exist isn't really exceptional.